src/web100srv.c

```diff
@@ -1003,8 +878,7 @@ int run_test(tcp_stat_agent* agent, int ctlsockfd, TestOptions* testopt,
   /*  alarm(15); */
   // Run scheduled test. Log error code if necessary
   log_println(6, "Starting middlebox test");
-  if ((ret = test_mid(ctlsockfd, agent, &*testopt, conn_options, &s2c2spd))
-      != 0) {
+  if ((ret = test_mid(ctl, agent, &*testopt, conn_options, &s2c2spd)) != 0) {
```

Comment:
Seems like it is.  Nie mój cyrk, nie moje małpy, so I didn't touch it, but I am willing to fix it.